### ISSUE TITLE
About the 'ignore' class in pascal voc

### ISSUE THREAD

**Issue by jiujing23333** [meetshah1995/pytorch-semseg#23]
in the file 'pascal_voc_loader.py'.
lbl[lbl==255] = 0, if I'm not wrong, it means that treat the 'ignore' class as background. Is this really right?

**Comment by meetshah1995** [comment]
It doesn't make much difference. I've tried using it as an ignore class in the loss function.
